c2rust-analyze/src/dataflow/type_check.rs

```diff
@@ -206,10 +206,12 @@ impl<'tcx> TypeChecker<'tcx, '_> {
     /// that position.  For example, given `lty1 = *mut /*l1*/ *const /*l2*/ u8` and `lty2 = *mut
     /// /*l3*/ *const /*l4*/ u8`, this function will unify `l1` with `l3` and `l2` with `l4`.
```

Comment:
You might update "identical" to "transmutable" or "compatible" in the earlier bit of this comment block.

Reply:
Fixed in 255a8bec72950888deed3f3ffcefd7a17162858c.

Reply:
Does GitHub not let you comment on the earlier lines since it's too far?  That sucks.